fee_allocator/accounting/distribution.py

```diff
@@ -112,66 +101,36 @@ def re_distribute_incentives(
-        aura_incentives_pct = _aura_incentives / _total_incentives
-        # If aura incentives pct is approx equal to aura_vebal_share, we don't need to do anything
-        if math.isclose(aura_incentives_pct, aura_vebal_share, rel_tol=1e-03, abs_tol=1e-03):
-            print(f"Pool {pool_id} aura incentives pct is approx equal to aura_vebal_share, skipping...")
-            continue
-        # If aura incentives percentage is less than aura_vebal_share, we need to increase bal incentives
-        if aura_incentives_pct >= aura_vebal_share:
-            # Calculate how much we need to increase bal incentives
-            bal_incentives_to_increase = round(
-                _total_incentives * (aura_incentives_pct - aura_vebal_share), 2)
-            # Calculate how much we need to decrease aura incentives but not to go below min aura incentive
-            if _aura_incentives - bal_incentives_to_increase <= min_aura_incentive:
-                # If aura incentives are less than min aura incentive after decreasing, we need to calculate
-                # how much we need to decrease aura incentives to min aura incentive
-                aura_incentives_to_decrease = _aura_incentives - min_aura_incentive
-                # Decrease aura incentives to min aura incentive
-                incentives[pool_id]['aura_incentives'] -= aura_incentives_to_decrease
-                incentives[pool_id]['bal_incentives'] += aura_incentives_to_decrease
-            else:
-                # Increase bal incentives
-                incentives[pool_id]['bal_incentives'] += bal_incentives_to_increase
-                # Decrease aura incentives
-                incentives[pool_id]['aura_incentives'] -= bal_incentives_to_increase
-        # If aura incentives percentage is more than aura_vebal_share, we need to increase aura incentives
-        else:
-            # Calculate how much we need to increase aura incentives
-            aura_incentives_to_increase = round(
-                _total_incentives * (aura_vebal_share - aura_incentives_pct), 2)
-            # Only increase aura incentives if it's more than min aura incentive
-            if _aura_incentives + aura_incentives_to_increase < min_aura_incentive:
-                continue
-            # Increase aura incentives
-            incentives[pool_id]['aura_incentives'] += aura_incentives_to_increase
-            # Decrease bal incentives
-            incentives[pool_id]['bal_incentives'] -= aura_incentives_to_increase
+        if _data['aura_incentives'] < min_aura_incentive:
+            # Calculate incentives to redistribute
+            incentives_to_redistribute = _data['aura_incentives']
+            # Set incentives to redistribute to 0
+            incentives[pool_id]['aura_incentives'] = 0
+            incentives[pool_id]['bal_incentives'] += incentives_to_redistribute
+            debt_to_aura_market += incentives_to_redistribute
+
+    if debt_to_aura_market:
```

Comment:
If we moved some to aura, we have to give it back.   We split it 6 ways evenly and give it to the top 6 earning pools.

Reply:
This makes sense to me